packages/frontend/core/src/modules/app-tabs-header/views/app-tabs-header.tsx

```diff
@@ -154,6 +154,10 @@ const WorkbenchTab = ({
                 onContextMenu={() => {
                   onContextMenu(viewIdx);
                 }}
+                onAuxClick={e => {
+                  onCloseTab(e);
```

Comment:
already called stopPropagation in onCloseTab, and we might need something like `useCatchEventCallback`

Reply:
i think this might not be that helpful in some cases that a button only stops or prevents default when only some conditions meet.
making a new hook for it does not help too much.

Reply:
but in most of cases, we just wrap a callback and stop the propagation. a more complex case does require to do it manually

Reply:
and I have to say, "the most cases" takes up to 80% out of near 90 results found in the codebase

Reply:
I have no strong opinion on this. Added `useCatchEventCallback` according to your suggestion.